lint/verilator_waiver.vlt

```diff
@@ -85,3 +85,19 @@ lint_off -rule UNUSED -file "*/rtl/ibex_decoder.sv" -match "*instr_alu*"
 // ibex_core.cs_registers_i.mie_q
 // Issue lowrisc/ibex#212
 lint_off -rule UNOPTFLAT -file "*/rtl/ibex_cs_registers.sv" -match "*u_ibex_core.cs_registers_i.mie_q*"
+
+// We have some boolean top-level parameters in e.g. simple_system.sv.
+// When building with fusesoc, these get set with defines like
+// -GRV32M=1 (rather than -GRV32M=1'b1), leading to warnings like:
+//
+//   Operator VAR '<varname>' expects 1 bits on the Initial value, but
+//   Initial value's CONST '32'h1' generates 32 bits.
+//
+// This signoff rule ignores errors like this. Note that it only
+// matches when you set a 1-bit value to a literal 1, so it won't hide
+// silly mistakes like setting it to 2.
+
+lint_off -rule WIDTH -file "*/rtl/ibex_simple_system.sv"
```

Comment:
I think we want to keep verilator_waiver.vlt for waivers relating specifically to the Ibex RTL, waivers for things like simple_system should live elsewhere (e.g. under examples/simple_system), though I'm uncertain if multiple waiver files are supported.

Was the normal lint check flagging up issues in simple_system?

Reply:
quick answer here: yes, multiple waivers should work just fine.

Reply:
Ok, I'll change this.

Reply:
Hmm, does this mean we need a separate signoff file for ibex_riscv_compliance too? It's a little sad, because the same signoff (with a different filename) will appear in both.

Reply:
There are no strict rules. But it would be nice to keep the core ibex one separated from all the other ones. I wouldn't mind having all the other waivers in a shared file.

Reply:
Well I don't think we strictly need the lint waivers, the actual lint check (as used by the CI) only looks at Ibex RTL so presumably these waivers are only about suppressing warnings when you do a normal build?

Reply:
You're right that these won't cause a lint check to fail, but I believe that one should never see warnings when building stuff unless something has gone wrong. (Yes, I know EDA tools fail at this terribly, but that's no reason for us to be rubbish too! :-) )

Reply:
Fair enough, happy to do that.

A single shared file for everything that isn't core ibex RTL should be fine for now. We can revisit that decision if it starts getting messy.

How shall we name things perhaps:

* lint/verilator_waiver_rtl.vlt
* lint/verilator_waiver_other.vlt

With some comments at the top of each explaining what should be in the file?

Reply:
Cool, sounds good to me.